sdk/iot/Azure.Iot.Hub.Service/tests/SessionRecords/StatisticsClientTests/StatisticsClient_Device_SuccessfulResponse.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://azadhub.azure-devices.net/statistics/devices?api-version=2020-03-13",
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "Authorization": "Sanitized",
+        "User-Agent": [
+          "azsdk-net-Iot.Hub.Service/1.0.0-dev.20200724.1",
+          "(.NET Core 4.6.29017.01; Microsoft Windows 10.0.18363 )"
+        ],
+        "x-ms-client-request-id": "bc279b23ff0d6f7b33c20faa29eda999",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "Content-Length": "69",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Fri, 24 Jul 2020 17:31:08 GMT",
+        "Server": "Microsoft-HTTPAPI/2.0",
+        "Vary": "Origin",
+        "x-ms-request-id": "5501bbdb-b068-46e8-b6e7-e993a57e41bd"
+      },
+      "ResponseBody": {
+        "totalDeviceCount": 2,
+        "enabledDeviceCount": 2,
+        "disabledDeviceCount": 0
+      }
+    }
+  ],
+  "Variables": {
+    "IOT_HUB_CONNECTION_STRING": "HostName=AzadHub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=Kg==;",
+    "RandomSeed": "1529393222"
```

Comment:
Did you want to check in with this ?
HostName=AzadHub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=Kg==;"

Reply:
yes, these are recorded tests, we have sanitized this as you can see so the key is not exposed. 
Do you mean why it's specific to my own hub? that's how it works, look at my hub as a placeholder, whenever we run the test in playback, that value will be replaced with the config details of whoever is running the test.